src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

```diff
@@ -13,6 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """ PyTorch LayoutLMv2 model."""
+"""
+LayoutLMv2 depends on detectron2, torchvision and tesseract; passing doctests requires their installation. Run the
+following to install them: `python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'` `python -m
+pip install torchvision tesseract`
+"""
```

Comment:
That's not the model doc file. The model doc file is in `docs/source/en/model_doc/layoutlmv2.mdx`.

Reply:
But aren't the model doc files also public facing? 

Where in `docs/source/en/model_doc/layoutlmv2.mdx` should I put these lines? Also, I noticed that in the `docs/source/en/model_doc/layoutlmv2.mdx`, the usage examples are the old doctests. Should I update these too? Or is there a tool to automatically sync the doctests? 

I don't feel like I have a good mental model of how doctests, this `mdx` file and the live documentation on HF websites interact -- it'd be great if you could point me to resources to learn it!

Reply:
That file is what the [documentation of the model](https://huggingface.co/docs/transformers/model_doc/layoutlmv2) will show. So it's the most important one to fix. The docstrings of the forward pass you fixed are then present in the documentation of each model (same page). The docstring of the module is nowhere in the doc, so not useful for most users.